ssl/quic/quic_txp.c

```diff
@@ -2745,7 +2734,7 @@ static int txp_generate_for_el(OSSL_QUIC_TX_PACKETISER *txp,
 
     /* Stream-specific frames */
     if (a.allow_stream_rel && txp->handshake_complete)
-        if (!txp_generate_stream_related(txp, pkt, min_ppl, force_explicit,
+        if (!txp_generate_stream_related(txp, pkt, force_explicit,
```

Comment:
I wonder if there was any reason in future why we would need this explicit minimum padding? @hlandau 

If not, this really nicely cleans up some dead code.